lua/telescope/actions.lua

```diff
@@ -90,7 +98,6 @@ local function goto_file_selection(prompt_bufnr, command)
       a.nvim_win_set_config(preview_win, {style = ''})
     end
 
-    local original_win_id = picker.original_win_id or 0
```

Comment:
Removed because its unused. After removing `original_win_id`, `local picker` is also no longer needed.